packages/trackerless-network/src/logic/utils.ts

```diff
@@ -40,3 +41,15 @@ export const hexToBinary = (hex: string): Uint8Array => {
     }
     return Buffer.from(hex, 'hex')
 }
+
+export const areEqualUsers = (userId1: UserID, userId2: UserID): boolean => {
+    return areEqualByteArrays(userId1, userId2)
+}
+
+const areEqualByteArrays = (arr1: Uint8Array, arr2: Uint8Array): boolean => {
```

Comment:
This function can just be:
`return Buffer.compare(a, b) === 0`
as in #1786 [@streamr/utils](https://github.com/streamr-dev/network/blob/ecc46c1564c2896bcc4e9a5b6d7f0eeced299b88/packages/utils/src/binaryUtils.ts#L26)

Reply:
Fixed 👍